src/components/DistributionOnchain/TokenForm.tsx

```diff
@@ -32,8 +32,7 @@ export default function TokenForm() {
 
   const defaultRecipients = uniqueCredits?.map((credit: any) => ({
     address: credit.address,
-    percentAllocation:
-      uniqueCredits?.length === 1 ? 100 : 100 / uniqueCredits.length,
+    percentAllocation: credit.splitBps / 100,
```

Comment:
if there are `2 contract types` with the total `bps` for each as `100` (or some other number)
- does `/ 100` account for the dynamic total `bps` possible in the credits array?
- what would happen if we divided by total `splitBps` rather than `100`?